configure.ac

```diff
@@ -86,6 +86,7 @@ AC_CONFIG_FILES( \
   t/sharness.d/00-setup.sh \
   t/src/Makefile \
   t/osu-micro-benchmarks/Makefile \
+  t/mpibench/Makefile \
```

Comment:
I'll just note this here, though I don't have a strong feeling about it one way or the other.

Sometimes it is easier to build these test programs from `t/Makefile.am` instead of a Makefile per program. The reasons include less duplication of boilerplate (e.g. `AM_CPPFLAGS`), you don't need to add a line to `configure.ac` for each new test program, and `make -j` can take better advantage of parallelism.

if there are only a couple of programs or if it makes more sense to keep builds of these programs separate then the other way is fine, I just wanted to mention that the alternative (build `t/mpibench` from `t/Makefile.am`) could save a tiny bit of work.

Reply:
Hmm, yeah this mpibench makefile probably isn't justified.  Good point.  Let me see if I can fix that real quick.